source/includes/opts/acceptDisclaimer.rst

```diff
@@ -0,0 +1,10 @@
+
+Accepts disclaimers for the :ref:`embedded verifier
+<c2c-verifier-disclaimer>` and :ref:`Beta <c2c-beta-program>`
+features (when using {+c2c-beta-program-short+}).
```

Comment:
Can we add something here that describes that this flag is intended for use by customers that have already read the disclaimer?

Reply:
Fixed